### PR TITLE
Some very good and smart changes.

### PR DESCRIPTION
**Pull request by DHervochon**
I've made some changes to the code. Now I'm writing a description.

Here's a new line.

_**This line has markup.**_

😂😂😂

### CODE REVIEW

**Comment by DHervochon** [comment]
This is a comment.

With multiple lines.

File: cd_win.c [hunk 1]
```diff
@@ -53,6 +53,12 @@ static void CDAudio_CloseDoor(void)
 		Con_DPrintf("MCI_SET_DOOR_CLOSED failed (%i)\n", dwReturn);
 }
 
+if (dwReturn = mciSendCommand(0, MCI_OPEN, MCI_OPEN_TYPE | MCI_OPEN_SHAREABLE, (DWORD) (LPVOID) &mciOpenParms))
```
Comment (DimaMzk) [inline cd_win.c:56]: This looks good!

File: cd_win.c [hunk 2]
```diff
@@ -342,16 +348,6 @@ static void CD_f (void)
 		return;
 	}
 
-	if (Q_strcasecmp(command, "info") == 0)
-	{
-		Con_Printf("%u tracks\n", maxTrack);
```
Comment (DimaMzk) [inline cd_win.c:347]: I don't know why this was removed, playing, and pausing music is a core part of the application

**Comment by DimaMzk** [comment]
These Changes look good!

🎉🎉🎉 🚀🚀🚀

**Comment by DHervochon** [comment]
This is a test

**Comment by DHervochon** [comment]
Another Test

**Comment by DimaMzk** [comment]
Quick test to make sure everything still works

**Comment by DimaMzk** [comment]
Quick test to make sure it all still works... Again

File: cd_win.c (hunk 2 above)
Comment (DimaMzk) [reply cd_win.c:347]: test

**Review by DimaMzk — APPROVED** [review]
This looks really good! I think you can go ahead an merge this in!

**Review by DimaMzk — APPROVED** [review]
This looks really good! I think you can go ahead an merge this in!

**Pull request merged**